app/app.ts

```diff
@@ -86,10 +93,40 @@ export class DoseAmigosApp {
                 // schedule an initial token refresh for when the
                 // token expires
                 this.auth.startupTokenRefresh();
+
+                // Notifications are scheduled/rescheduled every 5 minutes
+                this.nIntervId = setInterval(this.scheduleNotifications, 300000);
```

Comment:
This waits 5 minutes before making the first call. The first call should be immediate, then every 5 minutes.